nettacker/database/db.py

```
@@ -480,72 +1002,149 @@ def search_logs(page, query):
+                    parsed_event = json.loads(event)
+                    if parsed_event not in tmp["info"]["event"]:
+                        tmp["info"]["event"].append(parsed_event)
+                    parsed_json_event = json.loads(json_event)
+                    if parsed_json_event not in tmp["info"]["json_event"]:
+                        tmp["info"]["json_event"].append(parsed_json_event)
+
+                selected.append(tmp)
+            cursor.close()
+            connection.close()
+
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)